.eslintrc

```diff
@@ -0,0 +1,33 @@
+---
+
+root: true
+
+env:
+  browser: true
+  mocha: true
+  node: true
+
+rules:
+  comma-dangle: 0
```

Comment:
I don't have a super strong opinion at the moment, so disregard my comment for this PR. However, I think we'll have to remove this for compatibility reasons.